backend/auth/gateway/user/http/http_test.go

```
@@ -0,0 +1,127 @@
+
+func (f *fakeRegistry) ServiceAddress(ctx context.Context, serviceName string) (string, error) {
+	return f.addr, nil
+}
+
+func TestGetUserStatus_ReturnsStatusOnSuccess(t *testing.T) {
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != http.MethodGet || r.URL.Path != "/v1/user/user-123" {
+			t.Errorf("unexpected request: %s %s", r.Method, r.URL.Path)
+		}
+		w.Header().Set("Content-Type", "application/json")
+		json.NewEncoder(w).Encode(serviceresponse.Response[dto.GetUserStatusResponseDTO]{
```

golangci-lint (2.12.2)

[error] Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)